CLI/4. FAQs/2. TWCC CLI - COS.md

```diff
@@ -0,0 +1,17 @@
+---
+title : FAQ-CLI-COS | en
+tags: FAQ, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/default %}
+
+# TWCC FAQs | TWCC CLI: Cloud Object Storage (COS)
+
+:::spoiler Q1. Why do I fail when uploading and downloading a large number of files using TWCC-CLI?
+
+:::info
+Currently TWCC CLI file management function is still developing and optimizing. If you need to transfer a large number of files, see [<ins>this document</ins>](https://man.twcc.ai/@twccdocs/doc-cos-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Fcosbackup-en) to use s3cmd commands to operate.

```

Comment:
Currently TWCC CLI file management function is still developing and optimizing. > 

Currently, the TWCC CLI file management function is still developing and optimizing.

Reply:
If you need to transfer a large number of files, see [<ins>this document</ins>](https://man.twcc.ai/@twccdocs/doc-cos-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Fcosbackup-en) to use s3cmd commands to operate. >

To transfer a large number of files, refer to  [<ins>this document</ins>](https://man.twcc.ai/@twccdocs/doc-cos-main-en/https%3A%2F%2Fman.twcc.ai%2F%40twccdocs%2Fcosbackup-en)  to operate with s3cmd.